api/src/test/java/org/pmiops/workbench/ras/RasLinkServiceTest.java

```diff
@@ -124,22 +125,22 @@ private void mockAccessTokenResponse(String jsonResponse) throws IOException {
   }
 
   private long userId;
-  private Institution institution = new Institution();
+  private final Institution institution = new Institution();
   private static DbUser currentUser;
   private static List<DbAccessModule> accessModules;
 
   private RasLinkService rasLinkService;
-  private ObjectMapper objectMapper = new ObjectMapper();
+  private final ObjectMapper objectMapper = new ObjectMapper();
 
   @Autowired private UserService userService;
   @Autowired private UserDao userDao;
   @Autowired private AccessModuleDao accessModuleDao;
   @Autowired private UserAccessModuleDao userAccessModuleDao;
   @Autowired private AccessModuleService accessModuleService;
   @Mock private static OpenIdConnectClient mockOidcClient;
+
+  @Mock private static IdentityVerificationService mockIdentityVerificationService;
```

Comment:
would you mind adding a few "verify call to mock" / "verify no calls to mock" ID service here?

Reply:
Good call! I added a few of these.